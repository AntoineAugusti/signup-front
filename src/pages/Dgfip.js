import React from 'react';
import PropTypes from 'prop-types';
import Form from '../components/Form';
import Nav from '../components/Nav';
import DgfipFormConfiguration from '../components/form/config/dgfip';

const IntroDescription = () => (
  <div className="intro">
    <p>
      Dans le cadre du programme « Dites-le nous une fois », visant à simplifier
      les démarches administratives des usagers, l&apos;API Impôt Particulier
      permet de récupérer des informations fiscales des usagers de façon à leur
      éviter la transmission de leur avis d&apos;imposition papier.
    </p>
    <p>
      Ce portail permet de faciliter le raccordement du téléservice des
      fournisseurs de service à l&apos;API Impôt Particulier.
    </p>
    <p>
      Pour cela, il vous sera demandé de compléter le plus précisément possible
      les informations sur :
    </p>
    <ul>
      <li>le fondement juridique</li>
      <li>les données nécessaires à la démarche administrative</li>
      <li>la volumétrie de sollicitation de l&apos;API</li>
      <li>la protection des données personnelles.</li>
    </ul>
    <p>
      Pour faciliter votre racordement à l'API Impôt Particulier, une{' '}
      <a
        href="https://github.com/france-connect/service-provider-example/"
        target="_blank"
        rel="noopener noreferrer"
      >
        API de test
      </a>{' '}
      est à votre disposition.
    </p>
  </div>
);

const DemarcheDescription = () => (
  <div className="information-text">
    <p>
      C&apos;est la raison pour laquelle vous collectez des données
      personnelles, l&apos;objectif qui est poursuivi par le traitement que vous
      mettez en place. Par exemple, « télé-procédure permettant aux usagers de
      demander une aide au paiement de la cantine des collégiens » ou «
      télé-procédure de demande de bourses de lycée ».
    </p>
  </div>
);

const CguDescription = () => (
  <div className="information-text">
    <p>
      Votre raccordement à l&lsquo;API Impôt Particulier nécessite
      l&lsquo;acceptation de la convention d&lsquo;adhésion fixant vos
      engagements et ceux de la DGFIP et la DINSIC. <br /> Les liens ci-dessous
      vous permettront de visualiser la convention type ainsi que ses annexes.{' '}
      <br /> La convention générée à l&lsquo;issue de votre demande de
      raccordement contiendra l&lsquo;ensemble des éléments propres à votre
      situation. <br /> Cette convention sera publiée sur api.gouv.fr et sera
      accessible via vos identifiants FranceConnect.
    </p>
  </div>
);

const CadreJuridiqueDescription = () => (
  <div className="information-text">
    <p>
      Pour pouvoir bénéficier du raccordement à l&lsquo;API Impôt Particulier,
      le cadre légal et réglementaire des fournisseurs de service doit permettre
      à la DGFIP de transmettre des données fiscales à votre entité
      administrative.
    </p>
    <p>
      Il vous est donc demandé de préciser les références du fondement légal de
      votre droit à demander ces informations (délibération du conseil
      municipal, décret …).
    </p>
  </div>
);

const DonneesDescription = () => (
  <div className="information-text">
    <p>
      La loi informatique et libertés définit les principes à respecter lors de
      la collecte, du traitement et de la conservation de données personnelles.
    </p>
    <p>L’article 6 précise :</p>
    <ul>
      <li>
        3° [les données] sont adéquates, pertinentes et non excessives au regard
        des finalités pour lesquelles elles sont collectées et de leurs
        traitements ultérieurs ;
      </li>
      <li>
        4° Elles sont exactes, complètes et, si nécessaire, mises à jour ; les
        mesures appropriées doivent être prises pour que les données inexactes
        ou incomplètes au regard des finalités pour lesquelles elles sont
        collectées ou traitées soient effacées ou rectifiées ;
      </li>
    </ul>
    <p>
      Nous vous remercions de sélectionner uniquement les données strictement
      nécessaires à votre téléservice. Le non-respect du principe de
      proportionnalité vous expose vis à vis de la CNIL.
    </p>
    <p>
      Vous pouvez consulter le{' '}
      <a
        href="/docs/API_impots_particulier_template_corps_juridique_avec_annexes.pdf"
        target="_blank"
      >
        Contrat de Service Technique (CST)
      </a>{' '}
      pour voir le détail des données disponibles.
    </p>
    <p>
      Si N représente l&apos;année de la demande et que j&apos;ai besoin de
      l&apos;information la plus fraîche : si mon usager fait la demande avant
      le mois d&apos;Aout, il faut que je demande le N-2; si mon usager fait la
      demande après, je demande le N-1.
    </p>
  </div>
);

const Dgfip = ({
  match: {
    params: { enrollmentId },
  },
}) => (
  <div className="documentation">
    <Nav isDgfip={true} />
    <div className="main-pane">
      <Form
        enrollmentId={enrollmentId}
        form={DgfipFormConfiguration}
        isDgfip={true}
        IntroDescription={IntroDescription}
        DemarcheDescription={DemarcheDescription}
        CguDescription={CguDescription}
        CadreJuridiqueDescription={CadreJuridiqueDescription}
        DonneesDescription={DonneesDescription}
      />
    </div>
  </div>
);

Dgfip.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      enrollmentId: PropTypes.string,
    }),
  }),
};

Dgfip.defaultProps = {
  match: {
    params: {
      enrollmentId: null,
    },
  },
};

export default Dgfip;
