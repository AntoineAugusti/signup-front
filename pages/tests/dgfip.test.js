import ReactTestRenderer from 'react-test-renderer'
import Dgfip from '../../pages/dgfip'

require('../../lib/tests/local-storage') // eslint-disable-line import/no-unassigned-import

describe('pages | dgfip', () => {
  describe('render', () => {
    const props = {
      url: {
        query: {
          id: '1'
        }
      }
    }
    const renderer = ReactTestRenderer.create(<Dgfip {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
