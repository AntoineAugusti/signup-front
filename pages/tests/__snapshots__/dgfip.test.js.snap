// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | dgfip render should match snapshot 1`] = `
<div
  className="page"
>
  <header>
    <nav
      className="nav"
    >
      <div
        className="nav__container"
      >
        <a
          className="nav__link"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Accueil de signup.api.gouv.fr"
            className="nav__logo"
            src="https://api.gouv.fr/img/logo_api.svg"
          />
        </a>
        <ul
          className="nav__links"
        >
          <li>
            <a
              href="http://localhost:3022/oauth/authorize?client_id=82546188522214c3577d35c283ce8842786649b35a026a9d44908037a597f29b&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Foauth-callback&scope=&response_type=token&state="
            >
              Se connecter
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </header>
  <main>
    <div>
      
      <div
        className="documentation"
      >
        <nav
          className="side-pane"
        >
          <ul
            className="side-pane__menu"
          >
            <li
              className="text-center"
            >
              <a
                className="side-pane__link"
                href="http://dgfip.com"
              >
                <img
                  src="/static/images/logo-dgfip.png"
                  style={
                    Object {
                      "mixBlendMode": "multiply",
                    }
                  }
                />
              </a>
            </li>
            <li
              className="side-pane__title"
            >
              <h3>
                Votre demande
              </h3>
            </li>
            <li>
              <a
                className="side-pane__link"
                href="#legal"
              >
                Fondement juridique
              </a>
            </li>
            <li>
              <a
                className="side-pane__link"
                href="#donnees"
              >
                Choix des données
              </a>
            </li>
            <li>
              <a
                className="side-pane__link"
                href="#volumetrie"
              >
                Volumétrie
              </a>
            </li>
            <li>
              <a
                className="side-pane__link"
                href="#cnil"
              >
                Obligation CNIL
              </a>
            </li>
            <li>
              <a
                className="side-pane__link"
                href="#convention"
              >
                Convention
              </a>
            </li>
          </ul>
        </nav>
        <div
          className="main-pane"
        >
          <form
            onSubmit={[Function]}
          >
            <h1>
              Demande d'accès à l'API Impôts Particulier
            </h1>
            <p>
              Dans le cadre du programme « Dites-le nous une fois - Particuliers», visant à simplifier les démarches administratives des usagers, l'API « Impôt Particulier » permet de récupérer des informations fiscales des usagers de façon à leur éviter la transmission de leur avis d'imposition papier.
            </p>
            <p>
              Ce portail permet de faciliter le raccordement du téléservice des fournisseurs de service à l'API « Impôt Particulier ».
            </p>
            <p>
              Pour cela, il vous sera demandé de compléter le plus précisément possible les informations sur :
            </p>
            <ul>
              <li>
                le fondement juridique
              </li>
              <li>
                les données nécessaires à la démarche administrative
              </li>
              <li>
                la volumétrie de sollicitation de l'API
              </li>
              <li>
                la protection des données personnelles.
              </li>
            </ul>
            <p>
              Un outil, conçu par la DGFiP et la DINSIC, est par ailleurs mis à votre disposition pour pouvoir tester l'authentification SSL et concevoir vos bouchons .
            </p>
            <p>
              Ce portail permet de faciliter le raccordement du téléservice des fournisseurs de service à l'API « Impôt Particulier ».
            </p>
            <div>
              <input
                checked={false}
                disabled={false}
                id="checkbox-administration"
                name="enrollment.administration"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="label-inline"
                htmlFor="checkbox-administration"
              >
                Êtes-vous ou représentez-vous une administration publique (État, collectivités territoriales...) ?
              </label>
            </div>
            <div>
              <input
                checked={false}
                disabled={false}
                id="checkbox-france_connect"
                name="enrollment.france_connect"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="label-inline"
                htmlFor="checkbox-france_connect"
              >
                Avez-vous intégré le bouton FranceConnect ou êtes-vous éligible à son intégration ?
              </label>
            </div>
            <div>
              <input
                checked={false}
                disabled={false}
                id="checkbox-autorisation_legale"
                name="enrollment.autorisation_legale"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="label-inline"
                htmlFor="checkbox-autorisation_legale"
              >
                Possédez-vous un cadre légal permettant de solliciter des données fiscales auprès de la DGFiP ?
              </label>
            </div>
            <h1
              id="legal"
            >
              Fondement juridique
            </h1>
            <section
              className="information-text"
            >
              <p>
                Pour pouvoir bénéficier du raccordement à l‘API Impôts Particulier, le cadre légal et réglementaire des fournisseurs de service doit permettre à la DGFIP de transmettre des données fiscales  à votre entité administrative.
              </p>
              <p>
                Il vous est donc demandé de préciser les références du fondement légal de votre droit à demander ces informations (délibération du conseil municipal, décret …) ainsi que les informations relatives à votre téléservice.
              </p>
            </section>
            <div
              className="form__group"
            >
              <label
                htmlFor="fournisseur_de_service"
              >
                Sélectionnez le Fournisseur de Service FranceConnect pour lequel vous souhaitez un raccordement
              </label>
              <p>
                <a
                  className="button"
                  href="http://localhost:3001/users/auth/france_connect"
                >
                  Se connecter auprès de France Connect afin de récupérer mes démarches
                </a>
              </p>
              <select
                name="enrollment.fournisseur_de_service"
                onChange={[Function]}
              >
                <option>
                  -- sélectionnez une démarche --
                </option>
                
                <option
                  value="Une démarche de test"
                >
                  Une démarche de test
                </option>
              </select>
            </div>
            <div
              className="form__group"
            >
              <label
                htmlFor="description_service"
              >
                Décrivez brièvement votre service ainsi que l‘utilisation prévue des données transmises
              </label>
              <textarea
                disabled={false}
                id="description_service"
                name="enrollment.description_service"
                onChange={[Function]}
                rows="10"
                value=""
              />
            </div>
            <div
              className="form__group"
            >
              <label
                htmlFor="fondement_juridique"
              >
                Veuillez transmettre le fondement juridique sur lequel s’appuie votre demande
              </label>
              <textarea
                disabled={false}
                id="fondement_juridique"
                name="enrollment.fondement_juridique"
                onChange={[Function]}
                rows="10"
                value=""
              />
            </div>
            <h1
              id="donnees"
            >
              Choix des données
            </h1>
            <section
              className="information-text"
            >
              <p>
                La loi informatique et libertés définit les principes à respecter lors de la collecte, du traitement et de la conservation de données personnelles.
              </p>
              <p>
                L’article 6 précise :
              </p>
              <ul>
                <li>
                  3° [les données] sont adéquates, pertinentes et non excessives au regard des finalités pour lesquelles elles sont collectées et de leurs traitements ultérieurs ;
                </li>
                <li>
                  4° Elles sont exactes, complètes et, si nécessaire, mises à jour ; les mesures appropriées doivent être prises pour que les données inexactes ou incomplètes au regard des finalités pour lesquelles elles sont collectées ou traitées soient effacées ou rectifiées ;
                </li>
              </ul>
              <p>
                Nous vous remercions de sélectionner uniquement les données strictement nécessaires à votre téléservice. Le non-respect du principe de proportionnalité vous expose vis à vis de la CNIL.
              </p>
              <p>
                Vous pouvez consulter le 
                <a
                  href="/static/docs/cst-dgfip.pdf"
                  target="_blank"
                >
                  Contrat de Service Technique (CST)
                </a>
                 pour voir le détail des données disponibles.
              </p>
            </section>
            <div
              className="form__group"
            >
              <fieldset
                className="vertical"
              >
                <legend>
                  Sélectionnez vos jeux de données souhaités
                </legend>
                <div>
                  <input
                    checked={false}
                    disabled={false}
                    id="checkbox-scope_dgfip_RFR"
                    name="enrollment.scope_dgfip_RFR"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <label
                    className="label-inline"
                    htmlFor="checkbox-scope_dgfip_RFR"
                  >
                    DGFiP - Revenu Fiscal de Référence (RFR) et nombre de parts fiscales du foyer
                  </label>
                </div>
                <div>
                  <input
                    checked={false}
                    disabled={false}
                    id="checkbox-scope_dgfip_adresse_fiscale_taxation"
                    name="enrollment.scope_dgfip_adresse_fiscale_taxation"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <label
                    className="label-inline"
                    htmlFor="checkbox-scope_dgfip_adresse_fiscale_taxation"
                  >
                    DGFiP - Adresse fiscale de taxation au 1er janvier
                  </label>
                </div>
              </fieldset>
            </div>
            <h1
              id="volumetrie"
            >
              Volumétrie
            </h1>
            <section
              className="information-text"
            >
              <p>
                Connaitre les données relatives à la volumétrie et à la saisonnalité de votre téléservice nous permettra de vous offrir la meilleure qualité de service possible. En effet, cela permettra de prévenir les pics de charges et de transmettre ces informations aux fournisseurs de vos données.
              </p>
              <p>
                Conformément à notre charte, nous nous réservons le droit de réduire ou couper les appels autorisés au fournisseur de service.
              </p>
            </section>
            <div
              className="form__group"
            >
              <label
                htmlFor="nombre_demandes_annuelle"
              >
                Connaissez-vous le volume global annuel des demandes de votre téléservice ?
              </label>
              <input
                disabled={false}
                id="nombre_demandes_annuelle"
                name="enrollment.nombre_demandes_annuelle"
                onChange={[Function]}
                type="text"
                value=""
              />
            </div>
            <div
              className="form__group"
            >
              <label
                htmlFor="pic_demandes_par_heure"
              >
                Connaissez-vous le pic de charge (en nombre de demandes horaires) ?
              </label>
              <input
                disabled={false}
                id="pic_demandes_par_heure"
                name="enrollment.pic_demandes_par_heure"
                onChange={[Function]}
                type="text"
                value=""
              />
            </div>
            <div
              className="form__group"
            >
              <label>
                Connaissez-vous la répartition de la charge des demandes mensuelles (0 si le service est fermé) ?
              </label>
              <div
                className="form__group"
              >
                <div
                  className="date_input_row"
                >
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_jan"
                    >
                      Janvier
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_jan"
                      name="enrollment.nombre_demandes_mensuelles_jan"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_fev"
                    >
                      Février
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_fev"
                      name="enrollment.nombre_demandes_mensuelles_fev"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_mar"
                    >
                      Mars
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_mar"
                      name="enrollment.nombre_demandes_mensuelles_mar"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_avr"
                    >
                      Avril
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_avr"
                      name="enrollment.nombre_demandes_mensuelles_avr"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_mai"
                    >
                      Mai
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_mai"
                      name="enrollment.nombre_demandes_mensuelles_mai"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_juin"
                    >
                      Juin
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_juin"
                      name="enrollment.nombre_demandes_mensuelles_juin"
                      onChange={[Function]}
                      type="text"
                      value={undefined}
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_jui"
                    >
                      Juillet
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_jui"
                      name="enrollment.nombre_demandes_mensuelles_jui"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_aou"
                    >
                      Août
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_aou"
                      name="enrollment.nombre_demandes_mensuelles_aou"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_sep"
                    >
                      Septembre
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_sep"
                      name="enrollment.nombre_demandes_mensuelles_sep"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_oct"
                    >
                      Octobre
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_oct"
                      name="enrollment.nombre_demandes_mensuelles_oct"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_nov"
                    >
                      Novembre
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_nov"
                      name="enrollment.nombre_demandes_mensuelles_nov"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="date_input_col"
                  >
                    <label
                      htmlFor="nombre_demandes_mensuelles_dec"
                    >
                      Décembre
                    </label>
                    <input
                      disabled={false}
                      id="nombre_demandes_mensuelles_dec"
                      name="enrollment.nombre_demandes_mensuelles_dec"
                      onChange={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
            <h1
              id="cnil"
            >
              Obligation CNIL
            </h1>
            <section
              className="information-text"
            >
              <p>
                L’entrée en vigueur le 25 mai 2018 du règlement européen sur la protection des données supprime totalement le régime déclaratif.
              </p>
              <p>
                Les fournisseurs de services ont en particulier l’obligation de tenir un registre de leurs activités de traitement, d’encadrer les opérations sous-traitées dans les contrats de prestation de services, de formaliser des politiques de confidentialité des données et des procédures relatives à la gestion des demandes d’exercice des droits.
              </p>
              <p>
                La désignation d’un délégué à la protection des données - successeur du correspondant informatique et libertés (CIL) dont la désignation est aujourd’hui facultative - devient obligatoire pour les organismes et autorités publics, et donc pour les fournisseurs de services.
              </p>
            </section>
            <div
              className="form__group"
            >
              <label
                htmlFor="delegue_protection_donnees"
              >
                Délégué·e à la protection des données
              </label>
              <input
                disabled={false}
                id="delegue_protection_donnees"
                name="enrollment.delegue_protection_donnees"
                onChange={[Function]}
                type="text"
                value=""
              />
            </div>
            <div
              className="form__group"
            >
              <input
                checked={false}
                id="demarche_cnil"
                name="enrollment.demarche_cnil"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="label-inline"
                htmlFor="demarche_cnil"
              >
                Je déclare avoir accompli mes démarches CNIL en accord avec le règlement général de protection des données
              </label>
            </div>
            <h1
              id="convention"
            >
              Convention
            </h1>
            <section
              className="information-text"
            >
              <p>
                Votre raccordement à l‘API « Impôt Particulier » nécessite l‘acceptation de la convention d‘adhésion fixant vos engagements et ceux de la DGFIP et la DINSIC. 
                <br />
                 Les liens ci-dessous vous permettront de visualiser la convention type ainsi que ses annexes. 
                <br />
                 La convention générée à l‘issue de votre demande de raccordement contiendra l‘ensemble des éléments propres à votre situation. 
                <br />
                 Cette convention sera publiée sur api.gouv.fr et sera accessible via vos identifiants FranceConnect.
              </p>
            </section>
            <iframe
              height="800px"
              src="/static/docs/charte-fc.pdf"
              width="100%"
            />
            <div
              className="form__group"
            >
              <input
                checked={false}
                disabled={false}
                id="validation_de_convention"
                name="enrollment.validation_de_convention"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="label-inline"
                htmlFor="validation_de_convention"
              >
                Je valide la présente convention
              </label>
            </div>
            <div
              className="button-list"
            >
              <button
                className="button"
                id="submit"
                name="subscribe"
                type="submit"
              >
                Soumettre la demande
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </main>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="jsx-696190668 footer__logo"
      >
        <a
          className="jsx-696190668"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Logo api.gouv.fr"
            className="jsx-696190668"
            src="/static/images/logo-api.gouv.fr-white.png"
          />
        </a>
        <ul
          className="jsx-696190668 footer__social"
        >
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://twitter.com/BetaGouv"
              title="Twitter"
            >
              <img
                alt="Twitter"
                className="jsx-696190668 icon icon-twitter"
                src="/static/images/social/twitter.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://github.com/betagouv"
              title="Github"
            >
              <img
                alt="Github"
                className="jsx-696190668 icon icon-github"
                src="/static/images/social/github.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="mailto:contact@particulier.api.gouv.fr"
              title="Nous écrire un mail"
            >
              <img
                alt="Email"
                className="jsx-696190668 icon icon-mail"
                src="/static/images/social/email.svg"
              />
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="footer__links"
      >
        <li>
          <h2>
            particulier.api.gouv.fr
          </h2>
        </li>
        <li>
          <a
            href="https://status.particulier.api.gouv.fr"
          >
            Status des API
          </a>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
